schedule/containers/engines_and_tools_docker.yaml

```diff
@@ -18,16 +18,6 @@ conditional_schedule:
         - containers/docker_runc
       s390x:
         - containers/docker_runc
-  supported:
```

Comment:
The `supported` conditional schedule is fully remove but still present under schedule.